nixos/modules/services/x11/desktop-managers/xfce.nix

```diff
@@ -169,6 +175,9 @@ in
       xfce4-notifyd
     ];
 
-    security.pam.services.xfce4-screensaver.unixAuth = true;
+    security.pam.services.xfce4-screensaver.unixAuth = cfg.enableScreensaver;
+
+    # Enable screensaver by default if xfce is enabled
+    services.xserver.desktopManager.xfce.enableScreensaver = mkDefault true;
```

Comment:
Why not set it directly on the `enableScreensaver` as the default value instead of doing it here?

Remember this only applies if the main module is also enabled.

Reply:
So if I want to disable I can set to false without mkForce

Reply:
But you don't need `mkForce`.

This is why it is named as `default`. Because you can overwrite it.